File: dataset/human/python/file_2014.py
def main():
  from collections import Counter
  n = int(input())
  s = input()
  mod = 10**9+7
  c = Counter(s)
  cv = list(c.values())
  cl = len(cv)
  ans = 1
  for i in range(cl):
    ans *= (cv[i]+1)
  print((ans-1)%mod)
main()

File: dataset/human/c/file_2014.c
#include<stdio.h>

int main()
{
    int a, b, c;
    scanf("%d %d %d", &a, &b, &c);
    int ans = 0;
    if(b > c){
        ans = b+c;
    } else {
        ans += 2*b;
        c -= b;
        if(a >= c){
            ans += c;
        } else {
            ans += a+1;
            
        }
    }

    printf("%d\n", ans);
    return 0;
}
